### PR TITLE
Add csp-html-webpack-plugin@3.0.4 sub-dependencies

### PR DESCRIPTION
**Pull request by mt-micky**
- [ ] Are you the package author?
- [x] Downloaded, extracted, and reviewed the contents of the package tarball on the npm public registry.
- [x] Checked `README`, `LICENSE`, `COPYING`, and header comments for conflicting or additional licenses.
- [x] Searched for "license", such as with `fgrep -ir license $PACKAGE_DIRECTORY`.
- [x] Compared the license text found to standard language on <htts://spdx.org/licenses/>.
- [ ] Asked the package author about the applicable license.

Newest version of those packages include correct license in their `package.json` file

### CODE REVIEW

File: index.csv [hunk 1]
```diff
@@ -225,3 +225,5 @@ longest,MIT,1.0.1
 repeat-element,MIT,1.1.2
 trim,MIT,0.0.1
 css-value,MIT,0.0.1
+domutils,BSD-2-Clause,1.5.1
```
Comment (ljharb) [inline index.csv]: this also applies to v1.1.1, v1.1.2, v1.1.3, v1.1.4, v1.1.5, v1.1.6, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v1.4.2, v1.4.3, and v1.5.0 - can you add those entries?

File: index.csv [hunk 2]
```diff
@@ -225,3 +225,5 @@ longest,MIT,1.0.1
...
+css-select,BSD-2-Clause,1.2.0
```
Comment (ljharb) [inline index.csv]: v1.2.0 is declared as "BSD-like": https://unpkg.com/browse/css-select@1.2.0/package.json, however, the license file is unchanged between v2.1.0, where it's marked as BSD-2-Clause, and v1.0.0, and every version in between.

Thus, can you also add entries for v1.0.0, v1.1.0, v1.3.0-rc0, and v2.0.0?

**Review by ljharb — CHANGES_REQUESTED** [review]

**Comment by mt-micky** [comment]
Updated

**Review by ljharb — APPROVED** [review]

**Comment by kemitchell** [comment]
@fb55 we're working on an npm package with data about licenses for old versions of npm packages that don't have standardized license metadata in `package.json`. I believe `css-select` and `domutils` are yours. Mind confirming that we've got these right?

```csv
#package,license,versions...
domutils,BSD-2-Clause,1.1.1,1.1.2,1.1.3,1.1.4,1.1.5,1.1.6,1.2.0,1.2.1,1.2.2,1.3.0,1.4.0,1.4.1,1.4.2,1.4.3,1.5.0,1.5.1
css-select,BSD-2-Clause,1.0.0,1.1.0,1.2.0,1.3.0-rc0,2.0.0
```

**Comment by ljharb** [comment]
@kemitchell author confirmation is ofc preferred, but ftr i checked all of the relevant tarballs for license text in a SPDX-less version, that matched the license text in a version with a SPDX :-)

**Comment by kemitchell** [comment]
@ljharb gotcha. I'd be happy to publish with additions on that basis. But if we can get confirmation from the dev, so much the better.

**Pull request merged**